logging/src/commonMain/kotlin/io/spine/logging/LoggingDomain.kt

```diff
@@ -49,6 +57,8 @@ public annotation class LoggingDomain(public val name: String) {
  * If the logging domain is not defined for a class, logging statements for it
  * will not be prefixed. Otherwise, the prefix would be the name of the logging domain
  * in square brackets followed by a space.
+ *
+ * If the receiver is `null`, the prefix is empty.
  */
-public val LoggingDomain.messagePrefix: String
-    get() = if (name.isEmpty()) "" else "[$name] "
+public val LoggingDomain?.messagePrefix: String
+    get() = if (this?.name?.isEmpty() != false) "" else "[$name] "
```

Comment:
This one looks really weird :)

Reply:
It is but! It's the most concise way for checking with nullability and emptiness.